### DISCUSSION TITLE
Remove Azuracast Default Jingle

### DISCUSSION THREAD

**Discussion by countrymusicfy** [AzuraCast/AzuraCast#4564] [Q&A]
Hi, any suggestions on removing or replacing the default Azuracast jingle?

**Comment by Vaalyn** [comment]
This is answered in our documentation on this page: https://docs.azuracast.com/en/administration/docker#custom-default-track

**Reply by countrymusicfy** [reply]
Thank you, I re-called it being in the admin dashboard and did not see it in the docs

**Reply by countrymusicfy** [reply]
@Vaalyn I followed the instructions in documentation, still no change

**Comment by countrymusicfy** [comment]
version: '2.2'

services:
    stations:
        volumes:
                - /var/azuracast/jingle.mp3:/usr/local/share/icecast/web/error.mp3

**Comment by countrymusicfy** [comment]
ERROR: for stations  Cannot start service stations: OCI runtime create failed: container_linux.go:370: starting container process caused: process_linux.go:459: container init caused: rootfs_linux.go:59: mounting "/var/azuracast/jingle.mp3" to rootfs at "/var/lib/docker/overlay2/478bf502a77e698103bbaf16422717d8aaa0f9b941150c139813d98c25e78fc3/merged/usr/local/share/icecast/web/error.mp3" caused: not a directory: unknown: Are you trying to mount a directory onto a file (or vice-versa)? Check if the specified host path exists and is the expected type
ERROR: Encountered errors while bringing up the project.

**Comment by countrymusicfy** [comment]
takes the whole station down

**Comment by Vaalyn** [comment]
I've just checked on my test installation and the `/usr/local/share/icecast/web/error.mp3` file exists in my stations container.
Does the `/var/azuracast/jingle.mp3` exist at that location on your host machine?

The `docker-compose.override.yml` method still works on my test server on the latest rolling-release.

Regardless, I have to correct my last reply.
I forgot that we added an upload section for intros to the `Edit Mount Point` page...

Edit the mount point for which you want to change the intro and switch to the `Intro` tab. There you can upload the intro file for that mount point.

The documentation about custom intros is slightly outdated since it's missing the intro upload section for mount points and we'll need to update that.
Sorry for the confusion.

**Comment by countrymusicfy** [comment]
@Vaalyn thanks, the intro file to the mount point kind of works. the "current intro file" name does not display in mount points. the intro file does play, but it cuts out every 10 seconds.
